<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stackedWidget</class>
 <widget class="QStackedWidget" name="stackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrograPop</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="Inicio">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Bienvenido a PrograPop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_usuario">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_conectar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>153</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Lobby">
   <widget class="QLabel" name="Nombre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="Puntuacion">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_salas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="batchSize">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Salas de Juego</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>485</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Sala">
   <widget class="QLabel" name="nombresala">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>35</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre Sala</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Opcion1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Opcion3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Opcion2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Opcion4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion</string>
    </property>
   </widget>
   <widget class="QLabel" name="pregunta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pregunta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
